alacritty_terminal/src/event.rs

```diff
@@ -6,7 +6,7 @@ use crate::term::SizeInfo;
 
 #[derive(Clone, Debug, PartialEq)]
 pub enum Event {
-    DPRChanged(f64, (u32, u32)),
+    DPRChanged(f64),
```

Comment:
Could we move this to below `Message(Message)`? For no good reason other than it looking better. :)